### PR TITLE
Thread improvements

### PR DESCRIPTION
**Pull request by charlotte-zhuang**
1. Delete all unnecessary messages
2. Reduce time to delete messages

### CODE REVIEW

File: src/commands/threads/threadManager.js [hunk 1]
```diff
@@ -218,27 +236,34 @@ module.exports = new Command({
...
-              `\`${prefix}tm remove <thread id>\``,
-              'Removes a thread - alias (rm)'
-            )
-        );
+        message.channel
+          .send(
+            new Discord.RichEmbed()
+              .setColor('#ccffff')
+              .setTitle('Thread Manager')
+              .setDescription('Unrecognized paramter, try using:')
```
Comment (lupatrick13) [inline src/commands/threads/threadManager.js:253]: Typo!

**Review by lupatrick13 — APPROVED** [review]
Looks good!

File: src/commands/threads/threadManager.js (hunk 1 above)
Comment (evanugarte) [reply src/commands/threads/threadManager.js:253]: ^
```suggestion
              .setDescription('Unrecognized parameter, try using:')
```

File: src/util/threadMessage.js [hunk 2]
```diff
@@ -229,19 +242,26 @@ async function multipleThreadResults(threadID, message, threads, createMode) {
...
  *  type: 'message' | 'collector',
  *  message: Discord.Message?,
  *  collector: Discord.ReactionCollector?,
  * }>}
  */
-async function pagination(templateEmbed, message, threads, itemsPerPage = 5) {
+async function pagination(args) {
+  const { templateEmbed, message, threads } = args;
+  const itemsPerPage = args.itemsPerPage || 5;
+  const keepalive = args.keepalive || 60000;
```
Comment (evanugarte) [inline src/util/threadMessage.js:263]: if you wanted to keep everything on the same line you can do
```suggestion
  const { templateEmbed, message, threads, itemsPerPage = 5, keepalive = 60000 } = args;
```

from [Mozilla docs on destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Default_values_2)

**Review by evanugarte — APPROVED** [review]
Looks great

**Comment by evanugarte** [comment]
discord uses threads now so we no longer need this

charlotte remains the sce discord bot legend

**Pull request closed without merging**